Bio/Seq.py

```diff
@@ -228,7 +228,12 @@ def __len__(self):
         return len(self._data)  # Seq API requirement
 
     def __getitem__(self, index):  # Seq API requirement
-        """Return a subsequence of single letter, use my_seq[index]."""
+        """ Return a subsequence of single letter, use my_seq[index].
```

Comment:
Unwanted space before "Return"

Reply:
This is why the TravisCI style checks are failing,

```
$ flake8 Bio/
Bio/Seq.py:230:1: D210 No whitespaces allowed surrounding docstring text
```

See https://github.com/biopython/biopython/blob/master/CONTRIBUTING.rst for how to setup ``flake8`` with the relevant plugins.